pkg/ccl/cliccl/debug_backup.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 The Cockroach Authors.
+// Copyright 2021 The Cockroach Authors.
```

Comment:
I don't think we usually change the copyright date on file renames.

Reply:
oops. done.